common/cgroup/cgroup.go

```diff
@@ -282,11 +304,40 @@ func CreateCgroups(root string, subcgroup string, serviceNames []string) error {
 		flags = syscall.MS_NOSUID |
 			syscall.MS_NOEXEC |
 			syscall.MS_NODEV
-		if err := syscall.Mount("cgroup", cPath, "cgroup", flags, c); err != nil {
+		if err := syscall.Mount("cgroup", cPath, "cgroup", flags, c); err != nil && err != syscall.EBUSY {
 			return fmt.Errorf("error mounting %q: %v", cPath, err)
 		}
+	}
+
+	// 4. Create symlinks for combined controllers
+	symlinks := getControllerSymlinks(cgroups)
+	for ln, tgt := range symlinks {
+		lnPath := filepath.Join(cgroupTmpfs, ln)
+		if err := os.Symlink(tgt, lnPath); err != nil {
+			return fmt.Errorf("error creating symlink: %v", err)
+		}
+	}
+
+	return nil
```

Comment:
We do that only in the container cgroups, in RemountCgroupsRO. For the "host-cgroup-in-rkt-namespace" we don't remount it read-only.